Src/Plugins/Security/SecurePlaintextSecretsValidators/SEC101_044.NpmCredentialsValidator.cs

```diff
@@ -101,7 +101,9 @@ protected override IEnumerable<ValidationResult> IsValidStaticHelper(IDictionary
                     responseWithNoCredentials.StatusCode == HttpStatusCode.NotFound ||
                     responseWithNoCredentials.StatusCode == HttpStatusCode.NonAuthoritativeInformation)
                 {
-                    return ValidationState.NoMatch;
+                    message = $"The Uri {uri} is generic, no way to validate the credential with for account {id}";
```

Comment:
>generic [](http://example.com/codeflow?start=49&length=7)

What does it mean, the 'URI' is generic? This code path seems to suggest one possibility is the URI can't be found, how is that generic? 

Why does a return value of 'OK' mean it is too generic?

Are we properly describing this condition? Do we need separate error messages for each condition?

Why are we even fixing this? Your change makes this URI return 'unknown' which means we will validate it over and over again. Why? i.e., why isn't the status code conclusive? 

these don't look like transient conditions, they look permanent.